tool/tsh/common/tshconfig_test.go

```diff
@@ -476,3 +476,43 @@ func TestProxyTemplatesMakeClient(t *testing.T) {
 		})
 	}
 }
+
+func TestLocalProxyTLSConfig(t *testing.T) {
```

Comment:
nit: Personally I don't think it's necessary to include these two tests as they don't seem to offer a lot of value. But you are welcome to keep them too.

Reply:
I know, but I didn't want to add no tests at all 🙈 
Any ideas for better tests?

Reply:
> Is there an easy example on how to set that up in a test?

There are some examples of creating a `tc` here:
https://github.com/gravitational/teleport/blob/93be07558ec27250e53622277d4a6f156808ddc5/tool/tsh/common/tshconfig_test.go#L336

Note that the `cf` and `tc` created this way cannot be used for the TLS-with-cert-generation case anyway as there are no app certs. 

Alternatively, for simplicity, just use a `nil` for `tc`, if just testing for the TLS-with-provided-certs case.

Reply:
nit: I would suggest just removing these two tests =p